quantile_forest/_quantile_forest.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         """Initialize base quantile forest regressor."""
         init_dict = {
             (
-                "base_estimator" if sklearn_version < parse_version("1.2.0") else "estimator"
```

Comment:
Nit: This can be reformatted to remove the parenthesis and formatted as a single line, for consistency with the other dict items.